File: C/Listas/Lista 01/ex05.c
//Lê um número e mostra a terça parte deste número.
#include <stdio.h>

int main(void){
    int n1;
    float tercparte;
    printf("Digite o número que deseja saber sua terça parte: ");
    scanf("%d", &n1);
    tercparte=n1/3;

    printf("A terça parte de %d = %.2f",n1, tercparte);
}

File: Java/extra/Lista 01/src/ex05.java
// Lê um número e mostra a terça parte deste número.

import java.util.Scanner;
public class ex05 {
    public static void main(String[] args) {
        float num;
        try(Scanner leitor = new Scanner(System.in)){
            System.out.println("Digite o número: ");
            num = leitor.nextFloat();
        }
        System.out.format("A terça parte de %.1f = %.1f", num, terca_parte(num));
    }

    static float terca_parte(float a){
        return a/3;
    }
}
